packaging/debian/globus-scheduler-event-generator/debian/rules

```diff
@@ -27,8 +27,6 @@ configure-stamp:
 
 	dh_autoreconf
 
-	GLOBUS_VERSION=6.0 \
-	GRIDMAP=/etc/grid-security/grid-mapfile \
 	dh_auto_configure -- \
```

Comment:
How come you're not setting `GLOBUS_VERSION=6.2` here too?

Reply:
The configure script for globus-scheduler-event-generator does not mention GLOBUS_VERSION, so it does not need to be set. The globus-scheduler-event-generator.spec file for fedora/rhel does not set it. Removing it in debian/rules makes it consistent.